concordium-consensus/src/Concordium/GlobalState/Persistent/Bakers.hs

```diff
@@ -156,7 +156,8 @@ migratePersistentEpochBakers migration PersistentEpochBakers{..} = do
                 SomeParam $ P6.updateFinalizationCommitteeParameters migrationProtocolUpdateData
             StateMigrationParametersP6ToP7{} -> _bakerFinalizationCommitteeParameters
             StateMigrationParametersP7ToP8{} ->
-                SomeParam $ unOParam $ _bakerFinalizationCommitteeParameters
+                SomeParam $ unOParam _bakerFinalizationCommitteeParameters
```

Comment:
Don't we have a linter in the CI that should catch these things?

Reply:
I don't think the CI will complain about too many $'s